app/views/site/copyright.html.erb

```diff
@@ -38,21 +38,20 @@
 <% end %>
 
 <% I18n.with_locale @locale do %>
-  <%= tag.div :lang => @locale, :dir => t("html.dir") do %>
+  <%= tag.div :lang => @locale, :dir => t("html.dir"), :class => "richtext" do %>
     <p>
-      <%= t ".legal_babble.introduction_1_html",
-            :registered_trademark_link => tag.sup(link_to(t(".legal_babble.introduction_1_registered_trademark_html"),
-                                                          :anchor => "trademarks")),
-            :open_data => tag.i(t(".legal_babble.introduction_1_open_data")),
-            :odc_odbl_link => link_to(t(".legal_babble.introduction_1_odc_odbl"),
-                                      t(".legal_babble.introduction_1_odc_odbl_url")),
-            :osm_foundation_link => link_to(t(".legal_babble.introduction_1_osm_foundation"),
-                                            t(".legal_babble.introduction_1_osm_foundation_url")) %>
+      <%= t ".legal_babble.introduction_1_ohm_html",
+            :cc0_link => link_to(t(".legal_babble.introduction_1_cc0"),
+                                 t(".legal_babble.introduction_1_cc0_url")) %>
     </p>
     <p>
-      <%= t ".legal_babble.introduction_2_html",
-            :legal_code_link => link_to(t(".legal_babble.introduction_2_legal_code"),
-                                        t(".legal_babble.introduction_2_legal_code_url")) %>
+      <%= t ".legal_babble.introduction_2_ohm_html",
```

Comment:
I started using a new naming convention that appends `_ohm` to the paragraph number, to more clearly signal that the message – particularly the set of interpolation variables – is specific to this fork. This should avoid conflicts and errors more straightforwardly than fudging numbers like in #279.